From 2d16e7c656f5076d5704525568bbe28e51f88c8f Mon Sep 17 00:00:00 2001
From: Serhii Kupriienko
Date: Mon, 9 Oct 2023 10:48:39 +0300
Subject: [PATCH] Fix ray.tune import

---
 tune_sklearn/utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tune_sklearn/utils.py b/tune_sklearn/utils.py
index 5ab71f8..1ff339c 100644
--- a/tune_sklearn/utils.py
+++ b/tune_sklearn/utils.py
@@ -14,7 +14,7 @@ import inspect
 from ray.tune.logger import (Logger, JsonLoggerCallback, CSVLoggerCallback,
                              TBXLoggerCallback, LegacyLoggerCallback,
                              LoggerCallback)
-from ray.air.integrations.mlflow import MLflowLoggerCallback
+from ray.tune.integration.mlflow import MLflowLoggerCallback
 
 try:
     from ray.tune.stopper import MaximumIterationStopper
-- 
2.39.0

